src/index.ts

```diff
@@ -42,11 +49,11 @@ if (isSilent) {
 // Little sexy branding log ;)
 // tslint:disable:no-trailing-whitespace
```

Comment:
Can you please also remove these ts-lint lines since you fixed the trailing whitespace issue? 😉